### PR TITLE
Add lastname column to User DB table and update API logic to save,update and return this data.

### PR DESCRIPTION
**Pull request by k0staa**
In order to fix issue [https://github.com/getferdi/ferdi/issues/1040](https://github.com/getferdi/ferdi/issues/1040) I updated `User` db table and added `lastname`. I changed the controllers to persist this field. I also updated all views where this data can be shown (data,account). 
To finalize the fix, I also added a pull request in the ferdi repository -> [https://github.com/getferdi/ferdi/issues/927](https://github.com/getferdi/ferdi/issues/927)
Import `lastname` from Franz should also work theoretically (I checked Franz API) but the whole import still has a different problem -> https://github.com/getferdi/ferdi/issues/927 so I couldn't test it.

### CODE REVIEW

File: resources/views/dashboard/data.edge [hunk 1]
```diff
@@ -147,6 +150,9 @@
               <td>
                   {{ workspace.name }}
               </td>
+              <td>
+                  {{ workspace.lastname }}
```
Comment (kytwb) [inline resources/views/dashboard/data.edge:154]: This `td` and the corresponding `th` above have been added to the wrong section of the dashboard.

**Review by kytwb — CHANGES_REQUESTED** [review]
Thanks! I left a minor review regarding the dashboard, the rest looks good to me.

File: resources/views/dashboard/data.edge (hunk 1 above)
Comment (k0staa) [reply resources/views/dashboard/data.edge]: Shame on me ... I tried to be too fast. Fixed.
2021-02-07-101051_1920x1080_scrot

Comment (kytwb) [reply resources/views/dashboard/data.edge]: No shame, that's why we're doing the review.

**Review by kytwb — APPROVED** [review]

**Pull request merged**